SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Windows.Shared/Channel/SessionTracker.cs

```diff
@@ -96,26 +100,29 @@ private void HandleEnqueuingLog(object sender, EnqueuingLogEventArgs e)
                 {
                     return;
                 }
-                e.Log.Sid = _sid;
+
+                // Correlate current session only to logs not specifying timestamps.
+                // Crash before restart when Analytics was disabled at that time must not have session identifier.
```

Comment:
Can you explain how it knows it crashed with **Analytics disabled** by `timestamp == null`?
I can see the point that Crashes logs don't have `Timestamp` but what's the correlation between the module being disabled and timestamp?

Reply:
When Analytics is disabled, the sessionId is missing from crash (analytics set sessionId to null on the context).
Since crash specified its own timestamp, we skip sessionId correlation.
Thus the sessionId will be null as expected.
This if check is in Android too except it uses session history in that case, which we don't need for windows as we don't have c/c++ crashes.